dataplanes/vpp/build/Dockerfile.vpp

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:bionic
+ARG DEBIAN_FRONTEND=noninteractive
+ARG REPO=release
+RUN apt-get update
+RUN apt-get -y install curl
+RUN curl -s https://packagecloud.io/install/repositories/fdio/${REPO}/script.deb.sh |  bash
+RUN apt-get update
+RUN apt-get -y install vpp vpp-plugins gzip corekeeper
+RUN apt-get -y purge curl
+RUN apt-get -y clean
+RUN mkdir -p /tmp/dumps
+RUN sysctl -w debug.exception-trace=1 
+RUN sysctl -w kernel.core_pattern="/tmp/dumps/%e-%t"
+RUN ulimit -c unlimited
+RUN sysctl -w fs.suid_dumpable=2
+CMD ["/usr/bin/vpp","-c","/etc/vpp/startup.conf"]
```

Comment:
I assume this means you're using the default startup.conf for VPP, which is likely ok. We'll likely want to iterate on this eventually and put this in the repository.

Also note that if you are not setting poll-sleep-usec ([example found here](https://github.com/mestery/vpp-container-fun/blob/master/docker/allinone/startup1.conf#L11)), then this will use all your CPUs and make your laptop run hot if you're running it there. :)

Reply:
sure, but I found that defaults that come with vpp package were adequate to use for dataplane controller.